test/blueprints/Do.Test.Blueprints.Service/Singleton.cs

```diff
@@ -1,23 +1,11 @@
-﻿using Do.Core;
-using Do.Database;
+﻿using Do.Database;
 using Do.ExceptionHandling;
 
 namespace Do.Test;
 
-public class Singleton
+public class Singleton(TimeProvider _system, Func<Entity> _newEntity, ITransaction _transaction)
 {
-    readonly ISystem _system;
-    readonly Func<Entity> _newEntity;
-    readonly ITransaction _transaction;
-
-    public Singleton(ISystem system, Func<Entity> newEntity, ITransaction transaction)
-    {
-        _system = system;
-        _newEntity = newEntity;
-        _transaction = transaction;
-    }
-
-    public DateTime GetNow() => _system.Now;
+    public DateTime GetNow() => _system.GetUtcNow().DateTime;

```

Comment:
provide `GetNow()` as an extension for `TimeProvider` and return `GetLocalNow().DateTime` and use that method in place of `_system.Now` occurences